sentinelhub/areas.py

```diff
@@ -193,15 +194,34 @@ def __init__(
         :param split_shape: Parameter that describes the shape in which the area bounding box will be split.
             It can be a tuple of the form `(n, m)` which means the area bounding box will be split into `n` columns
             and `m` rows. It can also be a single integer `n` which is the same as `(n, n)`.
+        :param split_size: Parameter that describes the size of patches (in meters) into which the area bounding box
+            will be split.
+            It can be a tuple of the form `(width, height)` which means the area bounding box will be split into patches
+            of size (width, height). It can also be a single integer `size` which is the same as `(size, size)`.
         :param reduce_bbox_sizes: If `True` it will reduce the sizes of bounding boxes so that they will tightly fit
             the given area geometry from `shape_list`.
         """
-        self.split_shape = _parse_to_pair(split_shape, allowed_types=(int,), param_name="split_shape")
+        if (split_shape is not None) and (split_size is None):
+            self.split_shape = _parse_to_pair(split_shape, allowed_types=(int,), param_name="split_shape")
+            self.split_size = None
+        elif (split_shape is None) and (split_size is not None):
+            self.split_shape = None
+            self.split_size = _parse_to_pair(split_size, allowed_types=(int,), param_name="split_shape")
```

Comment:
do we have a sexier way of doing this?

I'd try to condense this information into an attribute of type
```python
self.split_params: Tuple[Literal["shape", "size"], Tuple[int, int]]
```

But it could be that the code gets uglier this way.

Reply:
I like the idea, but it doesn't really get prettier

Reply:
i like it this way, lets leave it in this shape